tools/upgrade

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
The same is here

Reply:
Oops!  I forgot to change that back after testing.  The scripts work with either Python now.